index.html

```diff
@@ -0,0 +1,457 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
```

Comment:
♻️ Great choice to use a list to arrange your links. However please note that some of them (portfolio and about) are not working correctly.

- [ ] Please fix the destination so it matches the actual section `id`. Since you are anchoring to an element's id using `#id` as the `href` attribute, it must match the `id` you gave to the desired destination section of the page. For example: when `<a href="#about-me'>` the about section should be something like `<section id="about-me">`